addon/lib/Feature.jsm

```diff
@@ -936,6 +1095,27 @@ class Feature {
           this.introPanelMessages[this.treatment],
           isIntroPanel
         );
+
+        let counter = this.treatment === "private" ?
+        this.state.blockedResources.get(win.gBrowser.selectedBrowser) :
+        Math.round(this.state.timeSaved.get(win.gBrowser.selectedBrowser) / 1000);
```

Comment:
For historical reasons, can you change `Math.round` to `Math.ceil`? I use `Math.ceil` to calculate the pageAction badge value (just rounding up so that fractions of a second show as 1s, which looks more effective than 0s). That way, there won't be a rounding error between the value recorded in the summary ping and the actual seen value on the pageAction button.

Reply:
For consistency, I changed them all to `Math.ceil`.